'PLATFORM_MODIFIER' defined twice on macOS

**OS:** macOS
**Version:** 0.0.8
**Commit/Build:** 1dae7f9f3255777e516cc999182cdca458a4451c

Recent refactoring seems to have introduced this macro error:

```
[ 91%] Building C object CMakeFiles/openrct2.dir/src/openrct2-ui/input/keyboard_shortcut.c.o
In file included from /Users/aaron/Projects/OpenRCT2/OpenRCT2/src/openrct2-ui/input/keyboard_shortcut.c:34:
/Users/aaron/Projects/OpenRCT2/OpenRCT2/src/openrct2-ui/input/KeyboardShortcuts.h:28:13: error: 'PLATFORM_MODIFIER' macro
      redefined [-Werror,-Wmacro-redefined]
    #define PLATFORM_MODIFIER CTRL
            ^
/Users/aaron/Projects/OpenRCT2/OpenRCT2/src/openrct2-ui/../openrct2/platform/platform.h:52:13: note: previous definition is here
    #define PLATFORM_MODIFIER CMD
            ^
1 error generated.
```

--- a/src/openrct2/platform/platform.h
+++ b/src/openrct2/platform/platform.h
@@ -26,12 +26,6 @@ typedef struct rct2_install_info rct2_install_info;
 #define MAX_PATH 260
 #endif
 
-#ifdef __APPLE__
-#define KEYBOARD_PRIMARY_MODIFIER KMOD_GUI
-#else
-#define KEYBOARD_PRIMARY_MODIFIER KMOD_CTRL
-#endif
-
 #define INVALID_HANDLE -1
 
 #define TOUCH_DOUBLE_TIMEOUT 300

--- a/src/openrct2/platform/platform.h
+++ b/src/openrct2/platform/platform.h
@@ -26,6 +26,12 @@ typedef struct rct2_install_info rct2_install_info;
 #define MAX_PATH 260
 #endif
 
+#ifdef __APPLE__
+#define KEYBOARD_PRIMARY_MODIFIER KMOD_GUI
+#else
+#define KEYBOARD_PRIMARY_MODIFIER KMOD_CTRL
+#endif
+
 #define INVALID_HANDLE -1
 
 #define TOUCH_DOUBLE_TIMEOUT 300
@@ -42,11 +48,6 @@ typedef struct rct2_install_info rct2_install_info;
 #define CTRL 0x200
 #define ALT 0x400
 #define CMD 0x800
-#ifdef __APPLE__
-    #define PLATFORM_MODIFIER CMD
-#else
-    #define PLATFORM_MODIFIER CTRL
-#endif
 
 typedef struct resolution {
     sint32 width, height;

